test/e2e/spec/eid_oidc_provider/auth_spec.js

```diff
@@ -0,0 +1,108 @@
+describe("eId OpenId: The authorization endpoint", function() {
+    
+    var chakram = require('/usr/local/lib/node_modules/chakram/lib/chakram.js');
+    var expect = chakram.expect;
+    var URL = 'https://' + process.env.BOILERPLATE_DOMAIN + '/api/openid';
+
+    var response_type = process.env.OIDC_RESPONSE_TYPE;
+    var scope = process.env.OIDC_SCOPE;
+    var client_id = process.env.OIDC_CLIENT_ID;
+    var state = '32os93klas8923hf9qpw98qhf9f2q2q093fu';
+    var nonce = '2389fhq3034ht84f809hj348t943w94fcpm4';
+    var redirect_uri = process.env.OIDC_REDIRECT_URI;
+
+    it("does not redirect when client_id missing or invalid", function() {
+        let params = 'response_type=' + response_type
+            + '&scope=' + scope
+            + '&client_id=1'
+            + '&state=' + state
+            + '&nonce=' + nonce
+            + '&redirect_uri=' + encodeURIComponent(redirect_uri);
+        var request = URL + '/authorize/?' + params;
+        return chakram.get(request, {followRedirect: false})
+            .then(function(respObj) {
+                expect(respObj).to.have.status(200);
```

Comment:
is the response status really 200 when there was an error? Is there a way we can fix that?